src/beamlime/applications/_nexus_helpers.py

```diff
@@ -258,8 +303,7 @@ def _is_monitor(group: NexusGroup) -> bool:
 
 def _initialize_ev44(module_spec: StreamModuleValue) -> NexusGroup:
     parent = module_spec.parent
-    if len(parent['children']) != 1:
-        raise ValueError('Group containing ev44 module should have exactly one child')
+    _validate_ev44_module_spec(module_spec)
```

Comment:
It seems excessive to validate the spec every time we are using it. Can't we make sure to validate the specs when they are constructed, then we don't have to sprinkle validation code here and there?

Reply:
Yeah... that's true. It's already checking in the `collect_streaming_modules`.

Reply:
@jokasimr  Done...!